src/why3_tactic.ml

```diff
@@ -180,7 +180,8 @@ let handle : Pos.popt -> Proof.proof_state -> sig_state -> string option ->
   (* Get the meta type of the current goal (with quantified context). *)
   let trm = !((Proof.Goal.get_meta g).meta_type) in
   (* Add the axiom to the current signature. *)
-  let a = Sign.add_symbol ss.signature Const (Pos.none axiom_name) trm [] in
+  let a = Sign.add_symbol ss.signature
+            Privat Const (Pos.none axiom_name) trm [] in
```

Comment:
This is not the way to indent a `let`-binding. Please, change to:
```
let a =
  Sign.add_symbol ss.signature Privat Const
    (Pos.none axiom_name) trm []
in
```
or make it fit on one line by defining `Pos.non axiom_name`.

Reply:
fixed in d7b99a6f1252f9d51d90833cc70c9ff9774a01b8